client/java/src/main/java/io/openlineage/client/circuitBreaker/TestCircuitBreaker.java

```diff
@@ -0,0 +1,43 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.client.circuitBreaker;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.NonNull;
+
+public class TestCircuitBreaker extends CommonCircuitBreaker {
```

Comment:
We use this for integration tests within Spark and Flink integration, so the class shall be part of a openlineage-java jar. I can rename it into `StaticCircuitCreaker`, because it closes/opens based on list of boolean values `true, false,true` provided.